build/Targets/Analyzers.Versions.targets

```diff
@@ -9,28 +9,42 @@
     <MicrosoftCodeAnalysisAnalyzersSemanticVersion>1.0.1</MicrosoftCodeAnalysisAnalyzersSemanticVersion>
     <MicrosoftCodeAnalysisAnalyzersPreReleaseVersion>beta1</MicrosoftCodeAnalysisAnalyzersPreReleaseVersion>
 
+    <SharedTestSemanticVersion>1.0.0</SharedTestSemanticVersion>
+    <SharedTestPreReleaseVersion>beta1</SharedTestPreReleaseVersion>
+
+    <RoslynDiagnosticsAnalyzersSemanticVersion>1.1.1</RoslynDiagnosticsAnalyzersSemanticVersion>
+    <RoslynDiagnosticsAnalyzersPreReleaseVersion>beta1</RoslynDiagnosticsAnalyzersPreReleaseVersion>
+
+    <ToolsSemanticVersion>0.0.0</ToolsSemanticVersion>
+    <ToolsPreReleaseVersion>beta1</ToolsPreReleaseVersion>
+  </PropertyGroup>
+
+  <!-- Version info for FxCop analyzer packages -->
+  <PropertyGroup>
+    <!-- Master FxCop analyzer package, which is just a collection of individual analyzer packages below. -->
+    <FxCopAnalyzersSemanticVersion>1.0.2</FxCopAnalyzersSemanticVersion>
+    <FxCopAnalyzersPreReleaseVersion>beta1</FxCopAnalyzersPreReleaseVersion>
```

Comment:
Note that this package has already been published during VS 2015 RTM - we are just publishing a newer package now.